engine/src/flutter/lib/ui/semantics.dart

```diff
@@ -1337,6 +1337,141 @@ class SemanticsFlags {
     hasRequiredState,
     isRequired,
   ]);
+
+  /// Convert flags to a list of string.
+  List<String> toStrings() {
+    final List<String> trueFlags = <String>[];
```

Comment:
Could make this MUCH tighter using https://dart.dev/language/collections#if-element

Reply:
thanks!